lib/TimezoneOffsetUtil.js

```diff
@@ -93,8 +98,12 @@ module.exports = function(timezone, mostRecent, changes) {
     for (var i = 0; i < sorted.length; ++i) {
       var change = sorted[i];
       if (i === 0) {
-        change.time = sundial.applyTimezoneAndConversionOffset(change.jsDate, timezone, conversionOffset).toISOString();
-        change.timezoneOffset = sundial.getOffsetFromZone(change.time, timezone);
```

Comment:
The old line 97 is the offending one, replaced by new line 106, using `mostRecent` as the input to sundial's `getOffsetFromZone`.